packages/utils/telemetry-utils/src/mockLogger.ts

```diff
@@ -378,9 +378,8 @@ export interface IMockLoggerExt extends ITelemetryLoggerExt {
 export function createMockLoggerExt(minLogLevel?: LogLevel): IMockLoggerExt {
 	const mockLogger = new MockLogger(minLogLevel);
 	const childLogger = createChildLogger({ logger: mockLogger });
-	Object.assign(childLogger, {
+	return Object.assign(childLogger, {
```

Comment:
Opportunistic improvement (doesn't affect the gist of the PR; it was caught during an intermediate state).